### ISSUE TITLE
Pressing tab key from test panel test does not move focus to run and debug buttons

### ISSUE THREAD

**Issue by sbland** [microsoft/vscode#162759]
Type: <b>Bug</b>

# Behaviour
## Expected vs. Actual

Should behave like source control panel.
Currently cannot focus on the `Run test` button without using the mouse.

## Steps to reproduce:

1. Go to the testing panel `>view: Show Testing`
2. Type something into search.
3. Press tab twice to go to a test in the list
4. Use the up and down arrows to focus a test
5. Press Tab again

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.10
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Venv
-   Value of the `python.languageServer` setting: Pylance

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```

languageServer: "Pylance"

linting
• pylintArgs: "<placeholder>"

testing
• pytestEnabled: true

```

</p>
</details>

Extension version: 2022.14.0
VS Code version: Code 1.71.2 (74b1f979648cc44d385a2286793c226e611f59e7, 2022-09-14T21:12:14.256Z)
OS version: Linux x64 5.4.0-126-generic snap
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (12 x 4370)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|0, 1, 1|
|Memory (System)|31.31GB (17.38GB free)|
|Process Argv|--no-sandbox --force-user-env --unity-launch --crash-reporter-id ebc65e20-a9f1-4bca-baba-a37d7a051f72|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
2144e591:30553903
pyindex848:30577860
40g7c324:30573242

```

</details>
